src/docs/platforms/javascript/angular.mdx

```diff
@@ -174,7 +277,7 @@ Sentry.init({
 angular.module("yourApplicationModule", ["ngSentry"]);
 ```
 
-In case you're using the CDN version or the Loader, Sentry provides a standalone file for every integration:
+If you're using the CDN version or the Loader, Sentry provides a standalone file for every integration:
```

Comment:
Is the Loader referring to Lazy Loading Sentry? If so, is it worth a link to that content?

Reply:
Good catch, this is actually an incorrect statement. You cannot use the loader with old Angular.